src/algebraic_geometry/projective_spectrum/scheme.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import algebraic_geometry.projective_spectrum.structure_sheaf
+import algebraic_geometry.Spec
+
+/-!
+# Proj as a scheme
+
+This file is to prove that `Proj` is a scheme.
+
+## Notation
+
+* `Proj`      : `Proj` as a locally ringed space
+* `Proj.T`    : the underlying topological space of `Proj`
+* `Proj| U`   : `Proj` restricted to some open set `U`
+* `Proj.T| U` : the underlying topological space of `Proj` restricted to open set `U`
+* `pbo f`     : basic open set at `f` in `Proj`
+* `Spec`      : `Spec` as a locally ringed space
+* `Spec.T`    : the underlying topological space of `Spec`
+* `spo g`     : basic open set at `g` in `Spec`
+* `A⁰ₓ`       : the degree zero part of localized ring `Aₓ`
```

Comment:
I don't see the definition of this notation below.

Reply:
This notation is only used in the documentation string and comment of function definitions etc.

Reply:
The reason I didn't introduce this as a notation is because `degree_zero_part` takes more argument than just an `x`

Reply:
done